partner-api.yaml

```diff
@@ -42,18 +42,25 @@ paths:
               examples:
                 Partner:
                   value:
-                    partnerId: 'ABC12'
-                    anrede: HERR
+                    partnerId: SZY92
+                    vorname: Test
```

Comment:
wieso hast du so viele Felder weggenommen? Sollte das Beispiel nicht vollständig sein?

Reply:
Partnerkennzeichen und Rechte sind ja in eigene Objekte mit eigenem Endpunkt ausgegliedert. Weiterhin scheinen wirklich 1 oder 2 Felder aus der PEX 1.2 nicht in die PEX 2.0 übernommen worden zu sein. IMO sind die Beispiele relativ vollständig. Kannst ja mal mit den Feldern der YAML vergleichen ... soviel mehr ist da nicht.

Reply:
Wenn man genau hinschaut, sind die Beispiele jetzt sogar sehr viel umfänglicher als vorher. Weiterhin gibt es nicht den Anspruch dass sie komplett vollständig sind, richtig?

Reply:
also du hast hier felder wie nachname, externaPartnerId, webseite, aufsichtsgehierde, registrierungsnummer entfernt. da frage ich mich wieso, denn sie waren ja vorher im Beispiel enthalten (zumindest stehen sie auf der linken seite)

Reply:
Das Beispiel ist eine reale Response, eines echten Aufrufs. In diesem waren diese Felder nicht gesetzt, daher fehlen sie im Beispiel. Mangels Zeit habe ich mir nicht die Mühe gemacht, die Stammdaten vorher vollständig zu pflegen. War für mich  erstmal :GEFN:.Nicht vergessen, wir hatten viel zu tun und wenig Zeit :)

Reply:
Ich sage ja nur, dass da aktiv felder gelöscht wurden. mangels zeit schon, beispiele sind wichtig, GEFR (good enough for release) aber das Ticket ist damit halt nicht abgeschlossen 🙂 

Daher habe ich ja auch approved